app/models/post.rb

```diff
@@ -1,2 +1,17 @@
 class Post < ApplicationRecord
+  belongs_to :user, foreign_key: :author_id
+  has_many :comments
```

Comment:
- [ ] Kindly note that a post belongs to author because you have the author column in your posts table which is correct. So please change line `2` to `belongs_to :author, class_name: 'User'`